05/rodne_cislo_finish.py

```diff
@@ -0,0 +1,45 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+while True:
+    # MONTHS 30 DAYS
+    months_30_women = []
+    months_30 = [4, 6, 9, 11]
+    for month_30 in months_30:
+        months_30_women.append(month_30 + 50)
+    months_30 = months_30 + months_30_women
+
+    # MONTHS 31 DAYS
+    months_31_women = []
+    months_31 = [1, 3, 5, 7, 8, 10, 12]
+    for mesic_31 in months_31:
+        months_31_women.append(mesic_31 + 50)
+    months_31 = months_31 + months_31_women
+
+    years = list(range(86, 100)) + list(range(00, 23))
+    only_numbers = []
+
+    ID = list(input('Napiš rodné číslo: '))
+    for number in ID:
+        if number.isdigit():
+            only_numbers.append(number)
+            if len(only_numbers) == 10:
```

Comment:
Na tento if by si sa mala dostať až v momente kedy máš spracované všetky znaky, ty k nemu pristupuješ po pridaní každého znaku. T.j. ak zadám na vstup číslo 111111111111111111111111111 tak mi to prejde, pretože v momente kedy mi na riadku 25 pridáš 10tu jednotku do only_numbers tak mi to do toho ifu spadne a vyhodnotí mi to ako úspech.